diff -Naur fife-0.3.2.orig/engine/SConscript fife-0.3.2/engine/SConscript
--- fife-0.3.2.orig/engine/SConscript	2010-07-05 21:36:41.000000000 +0200
+++ fife-0.3.2/engine/SConscript	2010-11-26 14:31:12.615822582 +0100
@@ -128,7 +128,8 @@
 								  SHLIBEMITTER = '')
 else:
 	sharedlib = env.SharedLibrary(target = fife_tgt,
-								  source = compilefiles)
+								  source = compilefiles,
+								  LINKFLAGS=['-Wl,-soname,libfife.so.SONAME'])
 
 #**************************************************************************
 #python library target
