diff -urN fife-0.3.2r2/build/linux2-config.py fife-0.3.2r2-patched/build/linux2-config.py
--- fife-0.3.2r2/build/linux2-config.py	2010-11-22 20:02:14.000000000 +1000
+++ fife-0.3.2r2-patched/build/linux2-config.py	2011-11-23 21:32:36.295272519 +1100
@@ -82,8 +82,8 @@
 			('Xcursor', '')]
 
 	# fedora hack
-	if 'fedora' in platform.platform():	
-		libs.insert(0,('boost_system', ''))
+#	if 'fedora' in platform.platform():	
+	libs.insert(0,('boost_system', ''))
 
 	if (opengl):
 		libs.append(('guichan_opengl', ''))
